### ISSUE TITLE
Version conflict: global/local git, git config

### ISSUE THREAD

**Issue by megies** [hbons/SparkleShare#1425]
Sorry if that's been asked before, I did not find an issue answering this problem.

I use an up-to-date git (`1.8.4`) residing in user-space, compiled from source. Sparkleshare (`1.1.0`, compiled from source) seems to be using the globally installed git (`1.7.10.4`; system is Linux, Debian wheezy).
Now, in my git config (`~/.gitconfig`) I have an option (namely `push.default = simple`) that is not recognized by the older git version installed under `/usr`, so all syncronizing efforts by SparkleShare result in errors.

Long story short:
Can I tell SparkleShare to use my local git? Or can I tell it to ignore my `~/.gitconfig`?
